node/src/BaseClient.ts

```diff
@@ -1079,8 +1086,8 @@ export class BaseClient {
      * ```
      */
     public async set(
-        key: string | Uint8Array,
-        value: string | Uint8Array,
+        key: GlideString | Uint8Array,
+        value: GlideString | Uint8Array,
         options?: SetOptions,
```

Comment:
Include the DecoderOption here too
```suggestion
        options?: SetOptions & DecoderOption,
```

Reply:
Update the command parameters and link to DecoderOption